lib/hex_web/web/templates/package.html.eex

```diff
@@ -50,6 +50,11 @@ build_tools = @current_release && @current_release.meta["build_tools"] || []
           </ul>
         </dd>
       <% end %>
+
+      <%= if @hexdocs_url do %>
+        <dt>Hexdocs</dt>
```

Comment:
Hum, actually hold on, just saw the line below. I think we may need another place for this.